tdrs-frontend/src/components/Home/Home.jsx

```diff
@@ -171,7 +177,52 @@ function Home() {
             handleChange={handleChange}
             handleBlur={handleBlur}
           />
-          {shouldShowSttComboBox && (
+          <div className="usa-form-group">
```

Comment:
Im no frontend guy so take my question with a grain of salt! Should this get moved into it's own component?

Reply:
yep! it absolutely should. there are a few idiosyncrasies with the radio input behavior that made me feel un-confident moving it over to its own component, but i captured that task and open question in #3015